dpbench/config/reader.py

```diff
@@ -213,6 +218,21 @@ def read_implementation_postfixes(
         config.implementations.append(implementation)
 
 
+def read_precision_dtypes(config: Config, precision_dtypes_file: str) -> None:
+    """Read and populate dtype_obj data types file.
+
+    Args:
+        config: Configuration object where settings should be populated.
+        precision_dtypes_file: Path to the configuration file.
+
+    Returns: nothing.
```

Comment:
`Returns` should be removed from docstring. Sphinx will treat it as a valid tag and generate the docstring with an entry for "Returns"

Reply:
Done